src/alert.sh

```diff
@@ -0,0 +1,22 @@
+# NOTE: src/commons.sh must be included before this file
+
+# Alerts command completion to the user. First argument is the alert code,
+# and second is the command string to be printed in visual notification.
```

Comment:
Let's start using some rules for this sort of comment. In some way started to use it in the "commons.sh" file, I would like to discuss it with you later. Anyway, how about comment this function like this:

```
# Alerts command completion to the user.
# 
# @parameter First argument is the alert code
# @parameter Second argument is the command string to be printed in visual notification
#
# NOTE: src/commons.sh must be included before this file
```

Also, write two or three phrases that add some detail about this function.